crypto/dso/dso_win32.c

```diff
@@ -51,7 +52,11 @@ static HINSTANCE LoadLibraryA(LPCSTR lpLibFileName)
         for (i = 0; i < len_0; i++)
             fnamw[i] = (WCHAR)lpLibFileName[i];
 
-    return LoadLibraryW(fnamw);
+#if defined(OPENSSL_SYS_WIN_CORE)
+    return LoadLibraryExW(fnamw, NULL, 0);
```

Comment:
the indent is wrong.

Reply:
FYI, @richsalz is talking about the preprocessor directive...